pxtblocks/plugins/comments/blockComment.ts

```diff
@@ -225,6 +225,18 @@ export class CommentIcon extends Blockly.icons.Icon {
         this.setBubbleVisible(this.bubbleVisiblity);
     }
 
+    // TODO: switch our custom comment position serialization
```

Comment:
Was it intentional to leave these as to do's for the time being? Does this mean comments will be broken?

Reply:
yes it's intentional and no they aren't broken. i already implemented a separate means of serializing the comment position in the data element for blocks and luckily that still works just fine; this todo is really just meant as a sort of code cleanliness thing since we might as well use the supported route.

main reason i didn't do it in this pr is because it will mean having to upgrade old projects that use my version of the serialization